internal/service/ec2/ec2_ami_copy.go

```diff
@@ -256,6 +256,11 @@ func resourceAMICopy() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"uefi_data": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
```

Comment:
```suggestion
				Type:     schema.TypeString,
				Computed: true,
```

Needs to be Computed-only as it can't be set on the copy, only read.

Reply:
I see, changed it :)